src/explore-education-statistics-admin/src/pages/release/components/ReleaseStatusChecklist.tsx

```diff
@@ -239,6 +239,14 @@ const ReleaseStatusChecklist = ({ releaseVersion }: Props) => {
               releaseRouteParams,
             )}#${releasePreReleaseAccessPageTabs.publicAccessList.id}`,
           };
+        case 'UnresolvedComments':
+          return {
+            message: 'The content has unresolved comments',
+            link: generatePath<ReleaseRouteParams>(
+              releaseContentRoute.path,
+              releaseRouteParams,
+            ),
+          };
```

Comment:
The robot tests in `release_status.robot` check these warnings in the Publishing checklist.

I think it would be a good idea to adjust that suite to add an unresolved comment to a content block and then test that the warning appears. 

There's already some text blocks set up by the suite that a comment could be left on.